pages/learn/summer-school/quantum-simulation-summer-school-2022.vue

```diff
@@ -0,0 +1,161 @@
+<script lang="ts">
+import { Course, Prerequisite } from 'constants/learnContent'
+import { Component } from 'vue-property-decorator'
+import CourseOverviewPage from '~/components/logic/CourseOverviewPage.vue'
+import { GeneralLink, StartLearningUrl } from '~/constants/appLinks'
+
+@Component({
+  head: () => ({
+    title: '2022 Qiskit global summer school on quantum simulation'
+  })
+})
+export default class SummerSchoolCoursePage extends CourseOverviewPage {
+  routeName: string = 'summer-school/intro-quantum-computing-quantum-hardware-2020'
```

Comment:
i think this `routeName` should probably be `summer-school/quantum-simulation-summer-school-2022`